### PR TITLE
Don't redirect user to a step when the redirection failed to avoid a redirection loop

### PR DESCRIPTION
**Pull request by brexis**

### CODE REVIEW

**Comment by sorich87** [comment]
No tests for this?

**Comment by brexis** [comment]
Done!

**Comment by sorich87** [comment]
Please resolve the conflicts.

File: build/js/bootstrap-tour.js [hunk 1]
```diff
@@ -453,16 +457,32 @@
       }
     };
 
-    Tour.prototype._isRedirect = function(path, currentPath) {
+    Tour.prototype._isRedirect = function(host, path, currentPath) {
+      var current_host;
+      if (host !== '') {
+        current_host = document.location.href.substr(0, document.location.href.lastIndexOf(document.location.pathname));
```
Comment (sorich87) [inline build/js/bootstrap-tour.js]: Why not use `document.location.hostname` ?

Comment (brexis) [inline build/js/bootstrap-tour.js]: `host` is use to redirect the step to a specific location.  `window.location.hostname` and `window.location.host` don't return the complete host with protocole.

**Pull request merged**